.buildbot_dockerfile_debian

```diff
@@ -19,4 +19,4 @@ RUN useradd -m -u ${CI_UID} ci && chown ${CI_UID}:${CI_UID} .
 ARG CI_RUNNER
 ENV CI_RUNNER=${CI_RUNNER}
 COPY --chown=${CI_UID}:${CI_UID} . .
-CMD sh -x .buildbot.sh
+CMD ["sh", "-x", ".buildbot.sh"]
```

Comment:
This doesn't seem necessary?

Reply:
Ah, I see why now. Ignore me!

Reply:
It fixes a docker warning and is a separate commit.